README.md

```diff
@@ -104,14 +116,19 @@ To connect to a cloud instance of CloudVision you must set the following variabl
 
 When syncing from CloudVision, this plugin will create new devices that do not exist in Nautobot. In order for this to work properly, you must provide the following default value sin the nautobot config file.
```

Comment:
Typo here unrelated to your PR: "value sin"

Reply:
Fixed. Thanks